### PR TITLE
TDL-16054 added code comments

### PR DESCRIPTION
**Pull request by namrata270998**
# Description of change
added code comments

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_google_sheets/schema.py [hunk 1]
```diff
@@ -184,12 +184,15 @@ def get_sheet_schema_columns(sheet):
...
             # Remove prior_header column_name
+            # stop scanning the sheet and break
             sheet_json_schema['properties'].pop(prior_header, None)
             LOGGER.info('TWO CONSECUTIVE SKIPPED COLUMNS. STOPPING SCAN AT: SHEET: {}, COL: {}, CELL {}1'.format(
                 sheet_title, column_name, column_letter))
             break
 
         else:
+            # skipped < 2 prepare `columns` dictionary with index, letter, column name, column type and 
+            # if the column is to be skiped or not for each column in the list
```
Comment (savan-chovatiya) [inline tap_google_sheets/schema.py:195]:
```suggestion
            # if the column is to be skipped or not for each column in the list
```

Comment (namrata270998) [reply tap_google_sheets/schema.py]: done

**Review by savan-chovatiya — APPROVED** [review]

**Review by karanpanchal-crest — APPROVED** [review]

**Review by KrisPersonal — CHANGES_REQUESTED** [review]
Is commenting only applicable for schema.py? Are there no other comments required for other code?

**Comment by namrata270998** [comment]
> Is commenting only applicable for schema.py? Are there no other comments required for other code?
The other files including this file also had covered code comments in the majority of the code. The schema.py file contained most of the logic of the tap. Hence, tried to incorporate more code comments for better readability and understanding.

**Review by KrisPersonal — APPROVED** [review]

**Pull request merged**